src/components/createScanDialog/createScanDialog.js

```diff
@@ -185,7 +185,7 @@ class CreateScanDialog extends React.Component {
     );
   }
 
-  renderConcurrentScans({ values, handleOnEvent }) {
+  renderConcurrentScans({ values }) {
```

Comment:
required:
lets restore this, we think it's the cause of not submitting values... probably need to restore the "mockEvent" pieces too